src/main.cpp

```diff
@@ -544,6 +543,12 @@ void setup()
 #endif
 #endif
 
+#if defined(M5STACK_UNITC6L)
+    pinMode(LORA_CS, OUTPUT);
```

Comment:
If we never change that, use the existing PIN_POWER_EN macro rather than defining a new one. c6l_init should be switched to the lateInitVariant mechanism used in src/platform/extra_variants rather than being called here

Reply:
It does get passed in to RadioLib, so should get controlled there. This is essentially the same code as runs for the T-deck/pro, t-lora-pager, and rp2040.